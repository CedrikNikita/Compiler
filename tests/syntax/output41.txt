test41.c:4:2 error: expected unqualified-id before ‘’
