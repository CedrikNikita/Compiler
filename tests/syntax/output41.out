wonderful
wadsd
what
LOL
type int
type char
type double
int main
  int a
  int b
  int n
             a
            /
          ==
         /
            \
             3
       &&
      /
             b
            /
         \
          <
            \
             7
    ||
          n
         /
      \
       >=
         \
          3
